.github/workflows/ci-cd.yaml

```diff
@@ -88,6 +88,18 @@ jobs:
       - name: trento checks ID sanity test
         run: python3 hack/id_checker.py
 
+  check-licenses:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Install WWHRD
```

Comment:
We don't really need to run this in 2 steps, right?
`check-licenses` already depend on `install-wwhrd`, so calling the 2nd would call the 1st by default

Reply:
you are right, I just thought that in case of failure of the install it would be more clear having a step specifically doing that.

Anyways, I have removed the step.